subprojects/execution/src/test/groovy/org/gradle/internal/execution/steps/TimeoutStepTest.groovy

```diff
@@ -28,8 +26,7 @@ import java.time.temporal.ChronoUnit
 
 class TimeoutStepTest extends StepSpec {
```

Comment:
We should specify the type parameter on `StepSpec` here as well.

Reply:
I intentionally left that off. The compiler does what is expected (infers the type of `context` to be `Context`) and not defining the context is actually what we mean here (as the step can work regardless of context type).

Reply:
I don't like having using raw types there, though I won't hold up the PR because of this.

Reply:
I've clarified further.

Reply:
👍